hr_infraction/security/ir.model.access.csv

```diff
@@ -0,0 +1,20 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_infraction_employee,access_hr_infraction,hr_infraction.model_hr_infraction,base.group_user,1,0,0,0
+access_hr_infraction_user,access_hr_infraction,hr_infraction.model_hr_infraction,hr.group_hr_user,1,0,0,0
+access_hr_infraction_entry,access_hr_infraction,hr_infraction.model_hr_infraction,hr_infraction.group_hr_infraction_entry,1,1,1,1
+access_hr_infraction_approve,access_hr_infraction,hr_infraction.model_hr_infraction,hr_infraction.group_hr_infraction_approve,1,1,1,1
+access_hr_infraction_action_user,access_hr_infraction_action,hr_infraction.model_hr_infraction_action,hr.group_hr_user,1,0,0,0
+access_hr_infraction_action_entry,access_hr_infraction_action,hr_infraction.model_hr_infraction_action,hr_infraction.group_hr_infraction_entry,1,1,1,1
+access_hr_infraction_action_approve,access_hr_infraction_action,hr_infraction.model_hr_infraction_action,hr_infraction.group_hr_infraction_approve,1,1,1,1
+access_hr_infraction_warning_user,access_hr_infraction_warning,hr_infraction.model_hr_infraction_warning,hr.group_hr_user,1,0,0,0
+access_hr_infraction_warning_entry,access_hr_infraction_warning,hr_infraction.model_hr_infraction_warning,hr_infraction.group_hr_infraction_entry,1,1,1,1
+access_hr_infraction_warning_approve,access_hr_infraction_warning,hr_infraction.model_hr_infraction_warning,hr_infraction.group_hr_infraction_approve,1,1,1,1
+access_hr_infraction_category_employee,access_hr_infraction_category,hr_infraction.model_hr_infraction_category,base.group_user,1,0,0,0
+access_hr_infraction_category_user,access_hr_infraction_category,hr_infraction.model_hr_infraction_category,hr.group_hr_user,1,0,0,0
+access_hr_infraction_category_manager,access_hr_infraction_category,hr_infraction.model_hr_infraction_category,hr.group_hr_manager,1,1,1,1
+access_hr_infraction_pm,access_hr_infraction,hr_infraction.model_hr_infraction,group_payroll_manager.group_payroll_manager,1,0,0,0
```

Comment:
If there is any code that depends on the payroll module it (and related group permissions) should be moved to a separate module: `hr_infraction_payroll`